lesson_19/homework_l19_task_02.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import pathlib
+import requests
+import subprocess
+import time
+
+from logging import config
+
+path_to_project = pathlib.Path(f"{pathlib.Path.home()}/Projects/AQA_course/")
+
+logging.config.fileConfig(f"{path_to_project}/logging_config.ini")
+logging.getLogger('sampleLogger')
+
+logging.info("Precondition step:")
+logging.info("Run the HTTP server.")
+server = subprocess.Popen(("python3", "app.py"), cwd="Application")
+time.sleep(1)
+logging.info("The HTTP server was started!!")
+
+logging.info("Verify \"POST\" request.")
```

Comment:
This methods need to be as test cases inside Test Suite class. Not as just declarative rows. Please fix it.